src/compiler/crystal/semantic/literal_expander.cr

```diff
@@ -293,7 +293,29 @@ module Crystal
     #
     #     String.interpolation("foo", bar, "baz", qux)
     def expand(node : StringInterpolation)
-      Call.new(Path.global("String").at(node), "interpolation", node.expressions).at(node)
+      pieces = node.expressions
```

Comment:
I thought about doing `node.expressions.dup` here, but mutating the string interpolation is fine because it's not used later on.

Reply:
It might be worth adding a comment in the code.

Reply:
Done, thanks!